<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCircleDialog</class>
 <widget class="QDialog" name="AddCircleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCircleDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>216</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelX">
      <property name="text">
       <string>Center X Pos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditX"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelY">
      <property name="text">
       <string>Center Y Pos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditY"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelRadius">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelVel">
      <property name="text">
       <string>Velocity</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pusButtonAdd">
      <property name="text">
       <string>Add Circle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditR"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditVel"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditX</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>lineEditR</tabstop>
  <tabstop>lineEditVel</tabstop>
  <tabstop>pusButtonAdd</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
